src/vue/align-editor/align-editor.vue

```diff
@@ -1,51 +1,35 @@
 <template>
   <div class="alpheios-alignment-editor-container" v-show="showAlignEditor">
+    <div class="alpheios-alignment-editor-header">

```

Comment:
the ALIGN_EDITOR_HEADING message is wrong. It says "Define Origin and Target texts" (i.e. same as the text editor heading

Reply:
Changed to "Define Alignment Groups"